comment/views.py

```diff
@@ -0,0 +1,71 @@
+from django.shortcuts import render
+from .models import Comment
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from post.models import Post
+from .serializers import CommentSerializer
+
+
+# Create your views here.
+
+class CommentCreateView(APIView):
+    def post(self, request):
+        author=request.user
+        post = request.data.get('post')
```

Comment:
```suggestion
        author = request.user
        post = request.data.get('post')
```
사소한 부분이지만 지켜줘야합니다! 아래에도 조금씩 보이네요